hedera-mirror-monitor/src/main/java/com/hedera/mirror/monitor/publish/PublishScenarioProperties.java

```diff
@@ -39,6 +39,10 @@ public class PublishScenarioProperties extends ScenarioProperties {
 
     private boolean logResponse = false;
 
+    // Maximum length of the transaction memo string
+    @Min(13) // 13 is the length of the memo timestamp
```

Comment:
```suggestion
    @Min(13) // 13 is the length of the memo timestamp
    @Max(100)
```

services limit transaction memo to be 100 bytes, longer memo will cause failure.

Reply:
Got it, I have added the `@Max(100)`